kommunicate/src/main/java/io/kommunicate/services/KmService.java

```diff
@@ -32,7 +34,9 @@
     private Context context;
     private KmClientService clientService;
     private KmAutoSuggestionDatabase autoSuggestionDatabase;
-
+    public static final String KM_SKIP_BOT = "skipBot";
+    public static final String KM_NO_ALERT = "NO_ALERT";
```

Comment:
@reytum this must be present in Applozic sdk, or possibly moving these to Applozic sdk might be a better place as it is related to notifications and not necessarily kommunicate.

Reply:
Yes, will move it to applozic SDK, but right now we need some constants for reference as applozic SDK is imported via gradle.